package.json

```diff
@@ -28,6 +28,7 @@
         "babel-plugin-transform-require-ignore": "^0.1.1",
         "box-annotations": "^2.3.0",
         "box-react-ui": "^22.7.0",
```

Comment:
Can we replace this now that box-ui-elements is a dependency? I recall we are only importing a single utility function.

Reply:
yeah, we can